test/unit/test_uop_symbolic.py

```diff
@@ -426,8 +433,7 @@ def test_div_mod_recombine(self):
   def test_div_mod_recombine_folded_mod(self):
     a = Variable("a", 0, 2)
     b = Variable("b", 0, 100)
-    with self.assertRaises(AssertionError):
-      self.helper_test_variable((31 * a + 1) % 30 + ((31 * a + 1) // 30) * 30, 1, 63, "((a*31)+1)")
+    self.helper_test_variable((31 * a + 1) % 30 + ((31 * a + 1) // 30) * 30, 1, 63, "((a*31)+1)")
     with self.assertRaises(AssertionError):
```

Comment:
if you are interested, i can add a $200 bounty and lock to you to fully fix this divmod case.